src/index.html

```diff
@@ -1,13 +1,87 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <div class="card__img">
+        <img
+          src="./images/imac.jpeg"
+          alt=""
+          class="card__img-imac"
+        >
+      </div>
+      <div class="card-info">
+
+        <div class="card-info__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <div class="card-info__number">
+          Код товара: 195434
+        </div>
+      </div>
+
+      <rating class="card__rating">
+        <div class="card__ratinng-stars">
+          <img
+            class="star"
```

Comment:
re-use stars block, made in the previous task. Don't hardcode stars img. Their activeness should change according to the block modificator card__ratinng-stars--1, card__ratinng-stars--2, etc